src/test/FuzzerImpl.cpp

```diff
@@ -957,40 +957,216 @@ struct SponsoredEntryParameters
+                                uint32_t optionFlags)
+        : SponsoredEntryParameters()
+        , mShortKey(shortKey)
+        , mNativeAssetAvailableForTestActivity(
+              nativeAssetAvailableForTestActivity)
+        , mOptionFlags(optionFlags)
     {
     }
 
-    int64_t const mAssetAvailableForTestActivity;
-    bool const mIssueAsset;
-};
+    constexpr AccountParameters(int shortKey,
+                                int64_t nativeAssetAvailableForTestActivity,
+                                uint32_t optionFlags, int sponsorKey)
+        : SponsoredEntryParameters(sponsorKey)
+        , mShortKey(shortKey)
+        , mNativeAssetAvailableForTestActivity(
+              nativeAssetAvailableForTestActivity)
+        , mOptionFlags(optionFlags)
+    {
+    }
 
-auto constexpr DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY = 256;
+    int const mShortKey;
+    int64_t const mNativeAssetAvailableForTestActivity;
+    uint32_t const mOptionFlags;
+};
 
 std::array<
     AccountParameters,
     FuzzUtils::NUMBER_OF_PREGENERATED_ACCOUNTS> constexpr accountParameters{
-    {{DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, false},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, true},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, true},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, true},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, true},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, false},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, false},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, false},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, false},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, false},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, false},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, false},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, false},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, false},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, false},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, false}}};
+    {{0, 256, 0},
+     {1, 256, 0},
+     {2, 256, 0, 1}, // sponsored by account 1
+     {3, 256, AUTH_REQUIRED_FLAG},
+     {4, 256, 0},
+     {5, 256, 0},
+     {6, 256, 0},
+     {7, 256, 0},
+     {8, 256, 0},
+     {9, 256, 0},
+     {10, 256, 0},
+     {11, 256, 0},
+     {12, 256, 0},
+     {13, 256, 0},
+     {14, 256, 0},
+     {15, 256, 0}}};
+
+struct TrustLineParameters : public SponsoredEntryParameters
+{
+    constexpr TrustLineParameters(int trustor, AssetID const& assetID,
+                                  int64_t assetAvailableForTestActivity,
+                                  int64_t spareLimitAfterSetup)
+        : TrustLineParameters(trustor, assetID, false, 0,
+                              assetAvailableForTestActivity,
+                              spareLimitAfterSetup)
+    {
+        assert(!mAssetID.mIsNative);
+    }
+
+    static TrustLineParameters constexpr WithAllowTrust(
```

Comment:
Nitpick: naming convention, should be `withAllowTrust` and analogous for the other ones

Reply:
Fixed.